android-components/components/service/contile/src/main/java/mozilla/components/service/contile/ContileTopSitesProvider.kt

```diff
@@ -24,7 +28,10 @@ import java.util.Date
 
 internal const val CONTILE_ENDPOINT_URL = "https://contile.services.mozilla.com/v1/tiles"
 internal const val CACHE_FILE_NAME = "mozilla_components_service_contile.json"
+internal const val CACHE_VALID_FOR_KEY = "valid_for"
+internal const val CACHE_TOP_SITES_KEY = "tiles"
 internal const val MINUTE_IN_MS = 60 * 1000
```

Comment:
🔍 nit: We should remote these and use DateUtils' `MINUTE_IN_MILLIS` and `SECOND_IN_MILLIS`